src/components/ProjectList/ProjectListContainer/projectListContainer.module.scss

```diff
@@ -1,11 +1,10 @@
 @import '../../../styles/variables.scss';
 .rtd {
-  text-align: right;
+  text-align: center;
 }
 .rtc {
   text-align: center;
   padding-top: 0;
```

Comment:
Should be able to combine these padding into one line just for cleanliness sake

Reply:
On it